### ISSUE TITLE
2-3 hours forecast interval in minimal view...

### ISSUE THREAD

**Issue by joernborup** [bramkragten/weather-card#27]
Hi..
First - thanks for a nice card for lovelace, looks very nice.

I would like to use your weather card in a setup where it takes a little real estate as possible but shows as much info as possible ;-) . The use case, is to have this card display at a tablet next the front door, so when the kids are leaving, they can see the weather now and for the next 3-8 hours for the day... when will it rain... do I need to  take a jacked ...

So you card, with OpenWeather as source (hourly forecast), and only shows forecast, gives a nice real estate.
But I'm missing:
-Time for the intervals, it just says the same day....
-how to include 'current' weather as the first column in the forecast ;-)

![image](https://user-images.githubusercontent.com/60480578/74614052-d00e1100-5114-11ea-99c7-6b6fb4c8b5c6.png)

/Regards Joern

**Comment by flouck** [comment]
Indeed, I agree with @joernborup, I see an hourly_forecast option as of today but it does not seem to work. Can you confirm? By the way, thank you @bramkragten for this great work, I love the aesthetic of your card!

**Comment by rafaelbiriba** [comment]
same issue for me!

**Comment by rafaelbiriba** [comment]
![image](https://user-images.githubusercontent.com/324743/81072947-35d59200-8ee7-11ea-8061-822fd1de8883.png)
![image](https://user-images.githubusercontent.com/324743/81072966-3e2dcd00-8ee7-11ea-86a7-fca5c44bee6f.png)

**Comment by advertisingfree** [comment]
yeah. i would like to see 2-hour forecast internal
